packages/experimental/Icon/src/Icon.tsx

```diff
@@ -16,7 +16,8 @@ function renderRasterImage(iconProps: IconProps) {
     rasterImageStyleCache({ width: width, height: height, tintColor: color }, [width, height, color])[0],
   );
 
-  return <Image source={iconProps.rasterImageSource.src} style={style} />;
+  const accessibilityLabel = iconProps.accessibilityLabel ?? 'Image';
```

Comment:
This string isn't localized, so I'd rather not have the default and instead do something like set the accessibilityRole to "image"

Reply:
Yeah +1 on not having any hardcoded non-localized strings

Reply:
Removed (Warren is OOF so I'm taking this over as it's needed urgently)